pkg/apis/certmanager/v1/types.go

```diff
@@ -110,7 +110,7 @@ const (
 	// as namespace/name.  The certificate is expected to have the is-serving-for annotations.
 	WantInjectAnnotation = "cert-manager.io/inject-ca-from"
 
-	// WantInjectAPIServerCAAnnotation, if set to "true", will make the cainjector
+	// WantInjectAPIServerCAAnnotation , if set to "true", will make the cainjector
```

Comment:
suggestion: I understand why you've done this, but simply adding a space here doesn't really look right to me. How about:

```suggestion
	// WantInjectAPIServerCAAnnotation will - if set to "true" - make the cainjector
```

Reply:
modified